sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,970 @@
+                        }
+                    ],
+                    "type": "panel",
+                    "breadcrumb": "default",
+                    "hideLabel": true,
+                    "tags": [],
+                    "conditional": {
+                        "show": "true",
+                        "when": "hasDependents",
+                        "eq": "yes"
+                    },
+                    "properties": {},
+                    "lockKey": true,
+                    "isNew": false
+                },
+                {
+                    "autofocus": false,
+                    "input": true,
+                    "tableView": false,
+                    "inputType": "radio",
+                    "label": "<strong>Are any of the dependants listed above supported by you financially but not under your sole custody?</strong>",
+                    "key": "supportnocustodyDependants",
+                    "values": [
+                        {
+                            "value": "yes",
+                            "label": "Yes",
+                            "shortcut": ""
+                        },
+                        {
+                            "value": "no",
+                            "label": "No",
+                            "shortcut": ""
+                        }
+                    ],
+                    "defaultValue": "",
+                    "protected": false,
+                    "fieldSet": false,
+                    "persistent": true,
+                    "hidden": false,
+                    "clearOnHide": true,
+                    "validate": {
+                        "required": true,
+                        "custom": "",
+                        "customPrivate": false
+                    },
+                    "type": "radio",
+                    "labelPosition": "top",
+                    "optionsLabelPosition": "right",
+                    "tags": [],
+                    "conditional": {
+                        "show": "true",
+                        "when": "hasDependents",
+                        "eq": "yes"
+                    },
+                    "properties": {},
+                    "hideLabel": false,
+                    "lockKey": true,
+                    "isNew": false
+                },
+                {
+                    "clearOnHide": false,
+                    "key": "dependantsCustodyPanel",
+                    "input": false,
+                    "title": "Dependents that you support financially but do not have sole custody",
+                    "theme": "default",
+                    "tableView": false,
+                    "components": [
+                        {
+                            "input": true,
+                            "tableView": false,
+                            "key": "dependantsSharedCustodyApplicationException",
+                            "label": "Application Exception - Dependant’s shared custody",
```

Comment:
How are the exceptions considered on the request for change perspective?? There is no workflow logic or consideration on this currently.

Can we check with business on this?

Reply:
It has been settled on the Teams discussion that these "exception" keys will be removed in this PR.